sympy/combinatorics/coset_table.py

```diff
@@ -765,12 +766,14 @@ def conjugates(self, R):
             R_set.difference_update(r)
         return R_c_list
 
+
 ###############################################################################
 #                           COSET ENUMERATION                                 #
 ###############################################################################
 
 # relator-based method
-def coset_enumeration_r(fp_grp, Y, max_cosets=None):
+def coset_enumeration_r(fp_grp, Y, max_cosets=None, resume=None,
```

Comment:
`resume` refers to an unfinished table. I think that a noun would be more suitable than a verb. Maybe something like `draft`.